front/lib/api/config.ts

```diff
@@ -176,6 +176,12 @@ const config = {
   getOAuthMicrosoftClientId: (): string => {
     return EnvironmentConfig.getEnvVariable("OAUTH_MICROSOFT_CLIENT_ID");
   },
+  getOAuthMicrosoftOutlookClientId: (): string => {
+    return EnvironmentConfig.getEnvVariable("OAUTH_MICROSOFT_OUTLOOK_CLIENT_ID");
+  },
```

Comment:
where is this used ? do we need a specific client id for outlook instead of microsoft_tools ?